packages/core/src/bar-chart.ts

```diff
@@ -251,7 +251,9 @@ export class BarChart extends BaseAxisChart {
 			.remove();
 
 		// Add slice hover actions, and clear any slice borders present
-		this.addDataPointEventListener();
+		if (this.options.tooltip.enabled) {
```

Comment:
it does because i've added it as the base option that it is enabled by default. So all the demo charts have it because they didnt override the base options. I can change it to this way if its not the same but its going to always be present in the options this way.